### PR TITLE
feat: Made images zoomable in SpeakerDetailsActivity

### LINKED ISSUE #2163: Make speakers image zoomable in SpeakerDetailsActivity
**Actual Behaviour**

The speaker's image is not zoomable.

**Expected Behaviour**

It should be zoomable.

**Would you like to work on the issue?**

Yes

**Parent issue**

#2098

### PR DESCRIPTION
**Pull request by dr0pdb**
Fixes #2163 

Changes: Made images zoomable in SpeakerDetailsActivity

Screenshots for the change:
videotogif_2018 01 19_01 42 00

@iamareebjamal Please review.

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java [hunk 1]
```diff
@@ -388,4 +391,16 @@ public void onOffsetChanged(AppBarLayout appBarLayout, int offset) {
...
+        String imageUri = Utils.parseImageUri(selectedSpeaker.getPhotoUrl());
+        if (imageUri != null) {
+            ZoomableImageDialogFragment zoomableImageDialogFragment = new ZoomableImageDialogFragment();
+            zoomableImageDialogFragment.setStyle(DialogFragment.STYLE_NO_TITLE, 0);
+            Bundle bundle = new Bundle();
+            bundle.putString(ConstantStrings.IMAGE_ZOOM_KEY, imageUri);
+            zoomableImageDialogFragment.setArguments(bundle);
+            zoomableImageDialogFragment.show(getSupportFragmentManager(), "ZoomableImage");
+        }
+    }
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java]: Perhaps we can make a Utility method for this code?

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by dr0pdb** [comment]
Changes: Added a utility class with a method for boilerplate code of showing a ZoomableImageFragmentDialog

@iamareebjamal Please review.

File: android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java [hunk 2]
```diff
@@ -388,4 +389,11 @@ public void onOffsetChanged(AppBarLayout appBarLayout, int offset) {
         }
     }
 
+    @OnClick(R.id.speaker_image)
+    public void onZoom() {
+        String imageUri = Utils.parseImageUri(selectedSpeaker.getPhotoUrl());
+        if (imageUri != null) {
+            ZoomableImageUtil.showZoomableImageDialogFragment(getSupportFragmentManager(), imageUri);
+        }
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java]: Null check can be handled inside the util method, also use TextUtils.isEmpty

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by dr0pdb** [comment]
Changes: Added empty check in Utils method using TextUtils.isEmpty.

@iamareebjamal Please review.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**